infobserve/common/queue.py

```diff
@@ -1,10 +1,27 @@
 import asyncio
+import pickle
+
+from aioredis import Redis
+
+from .logger import APP_LOGGER
+from .pool import RedisConnectionPool
 
 
 class ProcessingQueue:
+    REDIS_QUEUE = "redis"
+    SIMPLE_QUEUE = "simple"
 
-    def __init__(self, max_queue_size=0):
-        self.__queue = asyncio.Queue(max_queue_size)
+    def __init__(self, name, max_queue_size=0):
+        self.name = name
+
+        redis_pool = RedisConnectionPool()
+        if redis_pool.redis:
+            self.type = self.REDIS_QUEUE
+            self.__queue = redis_pool
+        else:
+            APP_LOGGER.warning("No redis configuration found initializing simple queue")
```

Comment:
`[...] configuration found. Initializing [...]`

Also I don't think this should be a warning. More so an `info`